pkgs/by-name/la/labwc/package.nix

```diff
@@ -77,7 +80,6 @@ stdenv.mkDerivation (finalAttrs: {
   meta = {
     homepage = "https://github.com/labwc/labwc";
     description = "Wayland stacking compositor, inspired by Openbox";
-    changelog = "https://github.com/labwc/labwc/blob/${finalAttrs.src.rev}/NEWS.md";
```

Comment:
why do we need to remove changelog?

Reply:
Parametric changelogs are discouraged because the link is fragile.

Reply:
and `url = "https://github.com/labwc/labwc/releases/tag/${finalAttrs.src.rev}/"` doesn't fit here?

Reply:
It would be ugly, and again, it can break when the revision is a raw hash.

Maybe a static link for Master branch is better.

Reply:
> It would be ugly, and again, it can break when the revision is a raw hash.
> 
> Maybe a static link for Master branch is better.

yep